samples/WebApiContrib.Core.Formatters.CsvFormatterSample/Controllers/CsvTestController.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using WebApiContrib.Core.Formatters.CsvFormatterSample.Model;
+
+namespace WebApiContrib.Core.Formatters.CsvFormatterSample.Controllers

```

Comment:
yeah that would be good, just `WebApiContrib.Core.Samples` or something like that